scripts/functions.ts

```diff
@@ -1 +1,90 @@
+import axios from "axios";
 
+import { IRepository } from "../models/Repository";
+
+const API_URL = "https://api.github.com/orgs/stackbuilders/repos";
+
+const getData = async (): Promise<any> => {
```

Comment:
This must return an array of a specific type for example:
```suggestion
const getData = async (): Promise<GithubData[]> => {
```